### DISCUSSION TITLE
Improvement to wallet UI for mobile use.

### DISCUSSION THREAD

**Discussion by nostitos** [btcpayserver/btcpayserver#5366] [Ideas / Feature Requests]
The plan is to add a link to my phone's home screen and use btcpay as my daily driver wallet.

**Add a scan QR code as part of the address field.**

Currently the component is there but it's at the very bottom of the page.

![image](https://github.com/btcpayserver/btcpayserver/assets/48504624/5ea6596e-0947-416f-b114-74d1c7a93b96)

Also change the title/suggested icon name from 
![image](https://github.com/btcpayserver/btcpayserver/assets/48504624/dcec7c60-cf8a-49de-a2cb-dac6e29965c5)

To the wallet's name first as title:
HotWallet or "Hotwallet Wallet" instead of "BTC Transactions".

**Comment by pavlenex** [comment]
Makes sense to move that one higher up. CC @dstrukt should we convert this one to an issue?
As for the title, imo should just be BTCPay Wallet, agreed it should be changed, nicely spotted.

**Comment by nostitos** [comment]
There's a lot of issues with the wallet,

Fee estimation is awful AND poorly defined.  Nobody seems to know why 1day estimate had a fee that is much lower than what should be required over more than a month.

The RBF - No RBF should be made to stick, Stick to random or stick to always RBF.
Some people pointed out that some nodes will relay to full-rbf miners. Well that's not happening! And the wallet doesn't let you try to do a full RBF.

Can't seems to add a label field to a URI (bip21) that will be saved anywhere in the wallet.

@dstrukt

**Reply by dennisreimann** [reply]
I agree that there's room for improvement and we already started to tackle those. The fixes in v1.12 include:

> Fee estimation is awful AND poorly defined

Fixed in #5490.

> Can't seems to add a label field to a URI (bip21) that will be saved anywhere in the wallet.

Fixed in #5428.

> And the wallet doesn't let you try to do a full RBF.

RBF is a bigger topic which we'll allocate more time to in v1.13

Thanks for the feedback!
